lib/spack/spack/__init__.py

```diff
@@ -3,10 +3,11 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-#: major, minor, patch version for Spack, in a tuple
-spack_version_info = (0, 17, 1)
+#: (major, minor, micro, dev release) tuple
+spack_version_info = (0, 18, 0, 'dev0')
```

Comment:
Is there any special meaning of "dev0" (or future plan to take advantage of the N there) or is it just to conform to the PEP spec that requires it?

Reply:
Just to confirm to it. This is the static module version which cant rely on git.

Reply:
The other reason to do it is if we want pip-installable dev versions -- like actual dev releases, as is the intent of PEP440.  We could do those periodically... more frequently than real releases.